chirp/drivers/retevis_rt22.py

```diff
@@ -332,7 +332,7 @@ def model_match(cls, data):
 
     if len(data) == 0x0408:
         rid = data[0x0400:0x0408]
-        return rid.startswith(cls.MODEL)
+        return rid.startswith(cls._model)
```

Comment:
Again, why duplicate that bytestring when you can easily do:

```
return rid.startswith(cls.MODEL.encode())
```
When we know the models are actual strings, a straight encode seems like the right thing to me. When they've got non-ASCII values in them, then it makes sense to have a different string, but I think in most cases you'll find they already do.

Reply:
I did this because in PR #54 you said that static strings should be defined as byte strings to avoid the runtime overhead of encoding them.

Reply:
That was different though. In that case we never needed to use the value as a string, and always needed it as `bytes`. Here, we've got a string that serves both purposes, so duplicating it is just risking one to get changed and not the other accidentally. The rule should be "if this was a string because py2 does not distinguish but is always used as bytes, then convert. If it's used in both forms, encode (or decode) as necessary to avoid the duplication/synchronization problem".

Reply:
This is getting fixed in #75 anyway, since a recent change to master fixed the syntax error.